4.figures/main_figure_6/scripts/main_figure_6.r

```diff
@@ -46,41 +57,41 @@ height <- 10
 options(repr.plot.width = width, repr.plot.height = height)
 
 umap_fig_gg <- (
-    ggplot(UMAP_results_df, aes(x = UMAP0, y = UMAP1))
-    + geom_point(
-            aes(color = Metadata_genotype),
-            size = 1.0,
-            alpha = 0.4
-    )
-    + theme_bw()
-    + guides(
-            color = guide_legend(
-                    override.aes = list(size = 5)
-            )
-    )
-    + labs(x = "UMAP0", y = "UMAP1", color = "NF1\ngenotype")
-    + facet_grid(factor(cell_line_derivative, levels = c("original", "derivative")) ~ ., 
-           labeller = labeller(.rows = function(x) paste("ipn02.3 2λ:", x)))
-    + coord_fixed(ratio = 0.52)
-    # change the text size
-    + theme(
-            strip.text = element_text(size = 17),
-            # x and y axis text size
-            axis.text.x = element_text(size = 22),
-            axis.text.y = element_text(size = 22),
-            # x and y axis title size
-            axis.title.x = element_text(size = 22),
-            axis.title.y = element_text(size = 22),
-            # legend text size
-            legend.text = element_text(size = 20),
-            legend.title = element_text(size = 22)
-    )
+        ggplot(UMAP_results_df, aes(x = UMAP0, y = UMAP1))
+        + geom_point(
+                        aes(color = Metadata_genotype),
+                        size = 1.0,
+                        alpha = 0.4
+        )
+        + theme_bw()
+        + guides(
+                        color = guide_legend(
+                                        override.aes = list(size = 5)
+                        )
+        )
+        + labs(x = "UMAP0", y = "UMAP1", color = "NF1\ngenotype")
+        + facet_wrap(~ factor(cell_line_code), 
```

Comment:
I like this change! What is missing, however, is the derivative vs. original nomenclature.

Will a `facet_grid(factor(cell_line_derivative, levels = c("original", "derivative")) ~ factor(cell_line_code))` do the trick? You'll also likely have to play around with the labeller as well.

Reply:
Ahh that is fair and was thinking about that... I like adding it as a facet. I will play around :D

Reply:
Got it! I have made the change and labeler works!